nsv13/code/modules/overmap/weapons.dm

```diff
@@ -211,6 +211,9 @@
 			return
 		fire_projectile(/obj/item/projectile/bullet/torpedo, target, homing = TRUE, speed=1, explosive = TRUE)
 		torpedoes --
+		var/obj/structure/overmap/OM = target
+		if(istype(OM, /obj/structure/overmap) && OM.dradis)
```

Comment:
Won't OM be null if target wasn't an overmap? Also you check for dradis twice.

Reply:
Nah, the left hand statement tries to cast it to be an overmap. If it isn't an overmap, then it'll still be registered as a var, but the statement will terminate because it's not passed the istype check